libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFManagedPreferences.m

```diff
@@ -63,6 +63,10 @@ + (instancetype)sharedPreferences {
     dispatch_once(&pred, ^{
         preferences = [[self alloc] init];
     });
+    
+    dispatch_once(&pred, ^{
```

Comment:
yes indeed. i have no clue how that worked the first time. this mean we need to change it in other places like SFuserAccountManager as well.